pack/config/BetterUI-ItemCounters.cfg

```diff
@@ -72,12 +72,12 @@ NoTier = 0
 [ItemScores]
 
 ## Score of each item for the ItemScore.
-## AACannon
+## ITEM_AACANNON_NAME
```

Comment:
I wonder what all these changes are

Reply:
yea idk, random shit seems to change when exporting profiles, i assume it was an update to the mod or something